public/admin/config.yml

```diff
@@ -99,6 +85,20 @@ collections:
           pattern: ["^.{50,160}$", "Must be between 50 and 160 characters"],
           i18n: true,
         }
+      - {
+          label: "Category",
+          name: "Category",
+          widget: "select",
+          options:
+            [
+              "admin-and-info",
```

Comment:
Could use un hyphonated strings as per given so content people don't have to think as much but if this is all wired up already, keep!

Reply:
Actually it is possible to configure the select to have Key:Value on these fields

Reply:
done